sirepo/srtime.py

```diff
@@ -44,15 +55,24 @@ def api_adjustTime(days=None):
     """
     from sirepo import http_reply
 
+    assert pkconfig.channel_in_internal_test(), \
+        'You can only adjust time in internal test'
```

Comment:
'API forbidden' is clearer, because that's what it is. I don't like idiomatic assert messages.